docs/jazzy.yml

```diff
@@ -181,11 +181,11 @@ custom_categories:
       - DistancedRoadObject
       - RoadObjectMatcher
       - RoadObjectsStore
-      - RoadObjectLocation
-      - RoadObjectType
+      - RoadObject.Location
+      - RoadObject.Kind
       - RoadObjectMatcherDelegate
       - RoadObjectsStoreDelegate
-      - RoadObjectEdgeLocation
+      - RoadObject.EdgeLocation
       - RoadObjectMatcherError
       - RoadObjectPosition
```

Comment:
This is the only other type that would make sense to rename to a nested type, for consistency with `RoadObject.Location`.

Reply:
Yep, was updated right after your comment.